src/components/Lottie/index.tsx

```diff
@@ -30,11 +33,40 @@ function Lottie({source, webStyle, ...props}: Props, ref: ForwardedRef<LottieVie
 
     const aspectRatioStyle = styles.aspectRatioLottie(source);
 
-    // If the image fails to load, app is in background state, animation file isn't ready, or the splash screen isn't hidden yet,
+    const browser = Browser.getBrowser();
+    const [hasNavigatedAway, setHasNavigatedAway] = React.useState(false);
+    const navigationContainerRef = useContext(NavigationContainerRefContext);
+    const navigator = useContext(NavigationContext);
+
+    useEffect(() => {
+        if (!browser || !navigationContainerRef || !navigator) {
```

Comment:
Do we need to check browser?

Reply:
Because the issue only occurs in browsers, it seems unnecessary to apply this fix to native platforms.

Reply:
OK, let change the PR status